### PR TITLE
Restore Force push in autobumper, unify build workflows with new naming convention

### PR DESCRIPTION
**Pull request by KacperMalachowski**
**Description**

Changes proposed in this pull request:

- Restore Force True in Push Config
- Fix paths in build workflows for image-autobumper
- ...

**Related issue(s)**

### CODE REVIEW

**Review by akiioto — APPROVED** [review]

**Comment by KacperMalachowski** [comment]
> Could you please describe why a force is needed and why you don't want to run test on edited activity?

Force is needed, because we don't want to include previous bumps in the history. The only what we want on bump PR is history from upstream's main and last bump commit.

Edited activity is not connected with code changes (for code the events is called `synchronize`) and we don't want to retrigger the tests upon PR description changes.

File: pkg/github/bumper/bumper.go [hunk 1]
```diff
@@ -127,13 +131,13 @@ func gitPush(remote, remoteBranch, baseBranch string, repo *git.Repository, auth
 	}
 
 	// Check if the remote head commit is the same as the local head commit
-	if remoteRef.Hash() == localRef.Hash() {
+	if remoteRefHash == localRef.Hash() {
```
[the remark this answers is not in the document]
Comment (KacperMalachowski) [reply pkg/github/bumper/bumper.go:134]: It doesn't work. Example: https://github.com/kyma-project/test-infra/actions/runs/11415911461/job/31766505934

With that change I allow to pass the check even if the remote reference doesn't exists as in example.
The whole check is only for preventing metadata-only updates that might cause test retrigger on bump PR.

[the remark this answers is not in the document]
Comment (dekiel) [reply pkg/github/bumper/bumper.go:134]: Can you add additional condition when checking err value returned from repo.Reference and do not introduce else block and new variable?

**Comment by KacperMalachowski** [comment]
> > > Could you please describe why a force is needed and why you don't want to run test on edited activity?
> > 
> > 
> > Force is needed, because we don't want to include previous bumps in the history. The only what we want on bump PR is history from upstream's main and last bump commit.
> > Edited activity is not connected with code changes (for code the events is called `synchronize`) and we don't want to retrigger the tests upon PR description changes.
> 
> The edited action description: "The title or body of a pull request was edited, or the base branch of a pull request was changed." If you have a PR with tests passed and you change a base branch will this workflow be triggered again for new target branch or not?

It won't, because the head of the PR will be the same

**Comment by KacperMalachowski** [comment]
Decided not to continue with current implementation. As `go-git` has some tricky way to replace `git push -f HEAD:remote-branch` command. I will research the topic more during implementation of similirar tool for one of my side-projects.

I created the follow up issue to implement completly custom bumper: https://github.com/kyma-project/test-infra/issues/12153. 
Restored original bumper from k8s team with required fixes in: https://github.com/kyma-project/test-infra/pull/12184

**Pull request closed without merging**